include/geos/profiler.h

```diff
@@ -80,8 +80,9 @@ class GEOS_DLL Profile {
 	void stop()
 	{
 		gettimeofday(&stoptime, nullptr);
-		double elapsed = 1000000*(stoptime.tv_sec-starttime.tv_sec)+
-			(stoptime.tv_usec-starttime.tv_usec);
+		double elapsed = static_cast<double>(
```

Comment:
Does it fix the warning if you end the literal number with a dot? Or what does the warning say exactly?

Reply:
The literal is not the problem, the problem is the whole computation.